back/app/channels/users_blocked_count_channel.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UsersBlockedCountChannel < ApplicationCable::Channel
+  def subscribed
+    stream_for 'users_blocked_count_channel'
+    ActionCable.server.broadcast 'users_blocked_count_channel', 'Subscribed!!'
```

Comment:
Seems like `subscribed` is never called (? websocket not really connected), as the `Subscribed!!` message broadcast is never seen in the logs, whereas the `Contact!!` message broadcast from `SideFxUserService#after_block` is.